client/components/accountoverview/accountOverviewCancelledCard.tsx

```diff
@@ -32,7 +32,8 @@ export const AccountOverviewCancelledCard = (
 
 	const showSubscribeAgainButton =
 		props.product.mmaCategory !== 'membership' &&
-		props.product.mmaCategory !== 'contributions';
+		props.product.mmaCategory !== 'contributions' &&
```

Comment:
should we add a todo to remove this?

Reply:
I've added a TODO comment above here and also a couple in the shared/productTypes.tsx file where grouped product type of contributions is referenced